### DISCUSSION TITLE
[Public roadmap]

### DISCUSSION THREAD

**Discussion by adeelassi** [community/community#29235] [Other Feature Feedback, Questions, & Ideas] [answered]
tiktok.com/@adeelassi_03 
I want to grow my account 
![Screenshot_20220812-015409_TikTok Lite](https://user-images.githubusercontent.com/111102969/184297189-3a3c0609-ba71-46a0-970c-4865d72dfc45.jpg)

**Comment by onurravli** [comment, marked as answer]
I don't think GitHub is the right place to advertise your TT account. :) Good luck.

**Reply by adeelassi** [reply]
Ok
